WordPress/Classes/ViewRelated/Stats/StatsViewController.m

```diff
@@ -13,9 +13,27 @@
 #import "WordPress-Swift.h"
 #import "WPAppAnalytics.h"
 #import "WPWebViewController.h"
+#import <WordPressComStatsiOS/WPStatsServiceCache.h>
 
 static NSString *const StatsBlogObjectURLRestorationKey = @"StatsBlogObjectURL";
 
+@interface WPStatsServiceCache (Shared)
+@end
+
+@implementation WPStatsServiceCache (Shared)
+
++ (nonnull instancetype)sharedCache
+{
+    static WPStatsServiceCache *cache = nil;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        cache = [[WPStatsServiceCache alloc] init];
```

Comment:
We often prefer to just use `new` instead of `alloc/init` since it is more succinct.